### PR TITLE
docs: update / remove duplicate content

### PR DESCRIPTION
**Pull request by syu-w**
Fixes: #405

### CODE REVIEW

**Comment by tigarmo** [comment]
here's the readthedocs build error:
```
/home/docs/checkouts/readthedocs.org/user_builds/canonical-rockcraft/checkouts/419/docs/explanation/overlay-step.rst:7: WARNING: unknown document: '/reference/parts'
/home/docs/checkouts/readthedocs.org/user_builds/canonical-rockcraft/checkouts/419/docs/how-to/install-slice.rst:74: WARNING: undefined label: 'chisel-example'
/home/docs/checkouts/readthedocs.org/user_builds/canonical-rockcraft/checkouts/419/docs/how-to/use-chisel.rst:6: WARNING: undefined label: 'chisel-example'
/home/docs/checkouts/readthedocs.org/user_builds/canonical-rockcraft/checkouts/419/docs/reference/rockcraft.yaml.rst:213: WARNING: undefined label: 'ref_parts'
/home/docs/checkouts/readthedocs.org/user_builds/canonical-rockcraft/checkouts/419/docs/tutorials/hello-world.rst:33: WARNING: undefined label: 'ref_parts'
```

**Review by tigarmo — APPROVED** [review]
Thanks! The linting error is about a line being too long in one of the rst files I think

File: docs/how-to/use-chisel.rst [hunk 1]
```diff
@@ -9,7 +9,27 @@ field without any ambiguities because slices follow an underscore-driven naming
...
+    chisel-openssl:
+      plugin: nil
+      stage-packages:
+        - openssl_bins
+        - ca-certificates_data
+
+    package-hello:
+      plugin: nil
+      stage-packages:
+        - hello
```
Comment (cjdcordeiro) [inline docs/how-to/use-chisel.rst:31]: I know this is coming from another page, but given that this is a guide, we can't have untested code blocks. This needs to go into the `code` folder to be tested by Spread.

**Review by cjdcordeiro — CHANGES_REQUESTED** [review]

File: docs/how-to/use-chisel.rst (hunk 1 above)
Comment (syu-w) [reply docs/how-to/use-chisel.rst]: After discussion, we decided to use the exists code for that example.

Comment (cjdcordeiro) [reply docs/how-to/use-chisel.rst]: ACK

**Review by cjdcordeiro — APPROVED** [review]

**Pull request merged**